build_environment.yml

```diff
@@ -16,7 +15,6 @@ dependencies:
   - fontconfig
   - freetype
   - gdal
-  - graphviz
```

Comment:
Can you check the jenkins logs, I'm pretty sure this provides the `dot` command which is needed for parts of the sphinx documentation.

Reply:
🤦‍♂ Ah I see, it is in the other yaml file.

Reply:
It is required for sphinx, but only when building the docs which jenkins_environment.yml handles.